__tests__/integration/nanocontracts/bet.test.ts

```diff
@@ -312,6 +312,7 @@ describe('full cycle of bet nano contract', () => {
     expect(txHistory).toHaveLength(4);
   };
 
+  /* eslint-disable jest/expect-expect -- All assertions are on the helper function */
```

Comment:
_Comment unrelated to line number_

You can accomplish the same thing, without disabling the rule by adding `expect.hasAssertions` in `beforeEach` of a shared jest file and adding `additionalTestBlockFunctions` as a parameter to the `jest/expect-expect` rule in the config, e.g.:

```json
{
  "rules": {
    "jest/expect-expect": [
      "error",
      { "additionalTestBlockFunctions": ["executeTests", "testLockedUtxoMethods"] }
    ]
  }
}
```

Reply:
The `additionalTestBlockFunctions` didn't work, but I managed to make the same improvement using `assertFunctionNames`.

Do you agree with this approach?